build/rakelib/genesis.rake

```diff
@@ -52,6 +57,17 @@ namespace :genesis do
     task :boot do
       system("sifnoded start --p2p.laddr tcp://0.0.0.0:26658 ")
     end
+
+    desc "Reset the state of a node"
+    task :reset, [:chainnet, :node_directory] do |t, args|
+      node_directory = if args[:node_directory].nil?
+        "~/.sifnoded"
```

Comment:
what is this doing?

Reply:
@utx0 that was just setting a default for the node_directory, however I've since changed this to be handled inside of `sifgen`.